docs/en/03_examples/00_elementalarea_with_dataobject.md

```diff
@@ -31,21 +27,15 @@ class BlogPostsAdmin extends ModelAdmin
 ```
 
 `$managed_models` - this static variable connects `BlogPostAdmin` with `BlogPost`, which will be created in the following code.\
```

Comment:
```suggestion
`$managed_models` - this static variable connects `BlogPostAdmin` with `BlogPost`, which will be created in the following code.
```

Reply:
If these are intended to be paragraph breaks, we should just have an empty line inbetween instead.

Reply:
Updatedu